packages/core/src/DdSdkReactNativeConfiguration.tsx

```diff
@@ -250,6 +252,18 @@ export class DdSdkReactNativeConfiguration {
      */
     public trackBackgroundEvents: boolean = DEFAULTS.trackBackgroundEvents;
 
+    /**
+     * Enables RUM for logs.
+     * By default, RUM is enabled for logs.
+     */
+    public bundleWithRum: boolean = DEFAULTS.bundleWithRum;
+
+    /**
+     * Enables Traces for logs.
+     * By default, Traces is enabled for logs.
+     */
+    public bundleWithTraces: boolean = DEFAULTS.bundleWithTraces;
+
```

Comment:
similar to my previous comment: user won't know to which component this applies without reading the doc. Yes, it is called `bundleWithRum`/`bundleWithTraces` in native SDKs, but there these methods are defined on the `Logger.Builder` instance, so it is clear that it applies to Logs.

Here we have these methods on the generic SDK configuration, so I think it is better to explicitly mention Logs in the naming, like `bundleLogsWithRum`/`bundleLogsWithTraces`.

I'm curious about the opinion of @louiszawadzki on that.

Reply:
Yes I agree this would make this more explicit by renaming these variables to `bundleLogsWithRum` / `bundleLogsWithTraces`.